### PR TITLE
ci: notify `nuxt/bridge` when composables change

### PR DESCRIPTION
**Pull request by wattanx**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
The composables in nuxt/bridge are ported from nuxt 3.
So, if there are changes in nuxt 3, we need to follow the changes.
In the `workflow` being added this time, if composables are modified in Nuxt 3, a notification will be sent to nuxt/bridge.

nuxt/bridge is already set up to receive notifications.
https://github.com/nuxt/bridge/pull/1013

(I would like to discuss if this name for the workflow is appropriate.)

### Todo
- [ ] Secrets must be prepared.
  - Need to add `contents: read & write` for nuxt bridge repo.
  - `metadata: read only` (automatically selected when selecting the contents permission)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/notify-nuxt-bridge.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: notify nuxt bridge
```
Comment (danielroe) [inline .github/workflows/notify-nuxt-bridge.yml:1]:
```suggestion
name: bridge
```

File: .github/workflows/notify-nuxt-bridge.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+on:
+  pull_request:
+    types: [closed]
+    paths:
+      - "packages/nuxt/src/app/composables/**"
+jobs:
+  notify-nuxt-bridge:
```
Comment (danielroe) [inline .github/workflows/notify-nuxt-bridge.yml:8]:
```suggestion
  notify:
```

**Review by danielroe — APPROVED** [review]
this looks 👌

**Comment by wattanx** [comment]
@danielroe 
Could you create a Personal Access Token (PAT) to be used in Actions?
(It is needed to add permissions to nuxt/bridge.)

- In the case of a fine-grained personal access token:
  - need `contents: read & write` and `metadata: read only` permissions.
- In the case of a classic PAT:
  - need `repo` scope 

https://github.com/peter-evans/repository-dispatch?tab=readme-ov-file#token

File: .github/workflows/notify-nuxt-bridge.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    if: github.event.pull_request.merged == true
+    strategy:
+      matrix:
+        repo: ["nuxt/bridge"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: repository dispatch
+        uses: peter-evans/repository-dispatch@bf47d102fdb849e755b0b0023ea3e81a44b6f570 #v2.1.2
+        with:
+          token: ${{ secrets.ACCESS_TOKEN }}
```
Comment (danielroe) [inline .github/workflows/notify-nuxt-bridge.yml:18]:
```suggestion
          token: ${{ secrets.BRIDGE_GITHUB_TOKEN }}
```

**Review by danielroe — APPROVED** [review]

**Pull request merged**